### ISSUE TITLE
508-defect-1 [COGNITION]: Contact Us page 711 link must have an accurate accessible label

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#17966]
#  [508-defect-1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear the TTY phone number read aloud accurately. Right now it reads just "seven" when I would expect it to read "seven one one".

## Details

Our TTY link needs the `aria-label` updated so it reads 711 properly. Code snippet and screen shot below.

## Acceptance Criteria

- [ ] The 711 aria-label is updated correctly
- [ ] Screen readers announce "Seven. One. One." correctly when the link has keyboard focus

## Solution (if known)

```diff
<a
  href="tel:711"
- aria-label="7"
+ aria-label="7 1 1."
>711</a>
```

## WCAG or Vendor Guidance (optional)

* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

## Screenshots or Trace Logs

![Screen Shot 2020-12-30 at 9 37 44 PM](https://user-images.githubusercontent.com/934879/103393656-71ae0f00-4ae9-11eb-81cc-f6ad7d49ae63.png)

**Comment by 1Copenut** [comment]
If this TTY link is being generated from the [Telephone component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/telephone/) you should be able to pass the prop `contact={CONTACTS['711']}` and the aria-label will render correctly. If it is being hard-coded, then please enter the aria-label as `aria-label="7 1 1."` and it will read correctly for assistive technology.

**Comment by bethpottsVADEPO** [comment]
Thank you @1Copenut ! I just fixed it.

**Comment by 1Copenut** [comment]
Thank you! I just verified the fix is in the network preview. Moving this item to close.
